lib/internal/bootstrap_node.js

```diff
@@ -474,6 +478,7 @@
   }
 
   NativeModule._source = process.binding('natives');
+  NativeModule.preloadModuleWrap = null;
```

Comment:
1. Personally I would set it to a null transform `a => a` and remove the if in line 552.
I like polymorphism 😊